config/base/numaflow-server/numaflow-server-metrics-proxy-config.yaml

```diff
@@ -178,4 +178,165 @@ data:
             - name: pod
               filters:
                 - name: pod
-                  required: false
\ No newline at end of file
+                  required: false
+    - name: mono_vertex_pod_cpu_memory_utilization
```

Comment:
can we merge monovertex and pipeline patterns for pod and containers, since it is repetitive?

Reply:
lets make it `Objects` which acts as a llist.